src/UI/Screens/EmailVerification.styles.ts

```diff
@@ -0,0 +1,101 @@
+import { StyleSheet } from "react-native";
+
+const styles = StyleSheet.create({
+
+    container: {
+        backgroundColor: '#FFFFE4',
```

Comment:
Put these hex codes and other such constants in a constant in file inside config folder for now.

Reply:
Already done sir